R/mail.R

```diff
@@ -0,0 +1,54 @@
+# Kaiaulu - https://github.com/sailuh/kaiaulu
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+
+
+
+#' Creates fake mbox data
+#'
+#' @param mlist specific mailing list associated with this mbox data
+#' @param reply_from email of sender
```

Comment:
please include a example as you did for reply_datetime here. It is borderline impossible for the user to guess the parameter otherwise. Just copy and paste from your unit test for the various parameters :)

Reply:
Also, I am not sure if this is quite right. Isn't this the author first name last name plus the email which I asked to be broken down into reply_from_author and reply_from_email on the parameters so your function combine them properly with the double quotes and the <> symbols?